### PR TITLE
Add greeting infra workflow

### PR DESCRIPTION
**Pull request by anludke**
Add greeting infra workflow in the documentation.
To merge after the following [PR](https://github.com/parodos-dev/serverless-workflows/pull/173) is merged.

### CODE REVIEW

File: content/docs/serverless-workflows/infrastructure/greeting.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Greeting
```
Comment (gciavarrini) [inline content/docs/serverless-workflows/infrastructure/greeting.md:2]: nit: Greeting Workflow?

Comment (anludke) [reply content/docs/serverless-workflows/infrastructure/greeting.md:2]: > nit: Greeting Workflow?

Other titles in https://github.com/parodos-dev/parodos-dev.github.io/tree/main/content/docs/serverless-workflows do not have "workflow" Let's keep it consistent.

**Comment by masayag** [comment]
I wouldn't add this workflow under that location.
If so, I'd add a section to the installation document "Test your installation" with steps to install the greeting workflow.
Therefore I'd recommend on closing this PR without merging it.

**Comment by anludke** [comment]
> I wouldn't add this workflow under that location. If so, I'd add a section to the installation document "Test your installation" with steps to install the greeting workflow. Therefore I'd recommend on closing this PR without merging it.

OK! I will create a ticket to go with your recommendation.

**Pull request closed without merging**